app/services/authorizer_cache.rb

```diff
@@ -9,6 +9,6 @@ def collection_cache_lookup(subject, permission)
     collection = @cache[subject.class.to_s][permission] ||=
       find_collection(subject.class, :permission => permission)
```

Comment:
`find_collection` is a very expensive function call, and it's worth caching it when we can.